Sources/Fluent/Model/Collection+Model.swift

```diff
@@ -0,0 +1,53 @@
+import Async
+
+extension Collection where Element: Model, Element.Database: TransactionSupporting {
+    /// Saves a collection of models to Database
+    ///
+    /// - Parameter conn: A means to connect to a database
+    /// - Returns: A future array of the saved models
+    func save(on conn: Element.Database.Connection) -> Future<[Element]> {
+        let databaseIdentifier = Element.defaultDatabase!
+        
+        return conn.transaction(on: databaseIdentifier) { (dbConn) -> EventLoopFuture<[Element]> in
+            return self.map { $0.save(on: dbConn) }.flatten(on: dbConn)
+        }
+    }
+    
+    /// Updates a collection of models in Database
+    ///
+    /// - Parameter conn: A means to connect to a database
+    /// - Returns: A future array of the updated models
+    public func update(on conn: DatabaseConnectable) -> Future<[Element]> {
+        let databaseIdentifier = Element.defaultDatabase!
+        
+        return conn.transaction(on: databaseIdentifier) { (dbConn) -> EventLoopFuture<[Element]> in
+            return self.map { $0.update(on: conn) }.flatten(on: conn)
+        }
+    }
+    
+    /// Creates a collection of models in Database
+    ///
+    /// - Parameter conn: A means to connect to a database
+    /// - Returns: A future array of the created models
+    func create(on conn: Element.Database.Connection) -> Future<[Element]> {
+        let databaseIdentifier = Element.defaultDatabase!
```

Comment:
I don't know in what situations `defaultDatabase` will be `nil`, so I'm not sure how folks want it handled.